### PR TITLE
Drop and obsolete rubygem-foreman_hooks

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
Can you open a Redmine issue? This also reminds me to revisit https://projects.theforeman.org/issues/37212.

**Comment by adamruzicka** [comment]
Opened https://projects.theforeman.org/issues/37295

**Comment by ekohl** [comment]
I think you missed a 2 in the commit message. I was surprised to see it a link there.

File: packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-Version: 1.6
+Version: 1.7
 Release: 1%{?dist}
 License: MIT
 Summary: A package to obsolete retired packages
 URL: https://github.com/theforeman/foreman-packaging
 BuildArch: noarch
 
 Obsoletes: rubygem-fog-google < 1.19.0-2
+Obsoletes: rubygem-foreman_hooks <= 0.3.17
```
Comment (ekohl) [inline packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec:10]: In other places we use the exact release + 1, so:
```suggestion
Obsoletes: rubygem-foreman_hooks < 0.3.17-4
```

This is a common practice to the point that I don't even know how `<=` operates in RPM.

**Review by ehelms — APPROVED** [review]

**Pull request merged**